docs/sources/integrations/available-integrations/configure-inbound-email/index.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Could you also please add OSS configuration instruction here? https://grafana.com/docs/oncall/latest/open-source/#email-setup